kafka/admin/client.py

```diff
@@ -948,6 +949,62 @@ def create_partitions(self, topic_partitions, timeout_ms=None, validate_only=Fal
                 .format(version))
         return self._send_request_to_controller(request)
 
+    def delete_records(self, records_to_delete, timeout_ms=None):
```

Comment:
it would be good to insert a new parameter with the value of the offset whose is the  parameter to estimate the smaller ones to delete.

`def delete_records(self, records_to_delete, offset_to_delete, timeout_ms=None):`